hack/tests/origin-conformance.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ex
+
+REAL="`realpath $0`"
+HERE="`dirname ${REAL}`"
+
+PATH=`go env GOPATH`/bin::$PATH
+TEST_SUITE="${TEST_SUITE:-openshift/conformance/parallel/minimal}"
+
+cleanup() {
+  set +e
+
+  generate_artifacts
+
+  delete
+}
+
+fetch_origin() {
+  local branch="${1:-release-3.11}"
+  local remote="https://github.com/openshift/origin.git"
+
+  test -d /tmp/origin || git clone --depth=1 --branch=${branch} $remote /tmp/origin
+}
+
+install_binaries() {
+  # install ginkgo
+  go get -v github.com/onsi/ginkgo/ginkgo
+
+  # build extended.tests and/or openshift-tests (regular user can't write to /usr/local)
+  make -C /tmp/origin build-extended-test && mv -v /tmp/origin/_output/local/bin/linux/`go env GOARCH`/* `go env GOPATH`/bin/
+
+  # install kubectl if not available
+  which kubectl || ( curl -s -L -o `go env GOPATH`/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/`go env GOARCH`/kubectl && chmod -v +x `go env GOPATH`/bin/kubectl )
+}
+
+function run-tests() {
+  if which openshift-tests && [[ -n "${TEST_SUITE}" ]]; then
```

Comment:
is openshift-tests a v4 only thing?  If so, remove references from here - solve the v3 problem, don't try to add code to solve a v4 problem here.

Reply:
Yes, openshift-tests is v4-only, removing all references to it to make it cleaner